3.Tests/UI/Hooks/Hooks.cs

```diff
@@ -41,12 +44,6 @@ public static void CleanUp()
         APIScripts.RemoveAllProjects();
     }
 
-    [AfterScenario]
-    public void SessionDisposal()
```

Comment:
There is no need to remove this hook, the CaptureScreenshot hook should only do that, not disposing the driver session as well.

Reply:
solved